darts/utils/data/sequential_dataset.py

```diff
@@ -0,0 +1,117 @@
+from typing import Union, Sequence, Optional
+from ...timeseries import TimeSeries
+from .timeseries_dataset import TimeSeriesDataset
+from ..utils import raise_if_not
+
+
+class SequentialDataset(TimeSeriesDataset):
+    def __init__(self,
+                 input_series: Union[TimeSeries, Sequence[TimeSeries]],
+                 target_series: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+                 input_length: int = 12,
+                 target_length: int = 1,
+                 max_samples_per_ts: Optional[int] = None):
+        """
+        A time series dataset containing tuples of (input, target) series, where "input" has length `input_length`,
+        and "target" has length `target_length`.
```

Comment:
Here I would add something like: 'In terms of time indices, "target" follows "input" without a gap. In other words, "target" starts one step after "input" ends.